src/controllers/connectionManager.ts

```diff
@@ -270,7 +270,7 @@ export default class ConnectionManager {
         listParams.ownerUri = connectionUri;
         listParams.includePassword = includePassword;
         const result = await this.client.sendRequest(ConnectionContracts.GetConnectionStringRequest.type, listParams);
-        return result.connectionString;
+        return result.toString();
```

Comment:
According to https://github.com/Microsoft/sqltoolsservice/blob/main/src/Microsoft.SqlTools.ServiceLayer/Connection/Contracts/GetConnectionStringRequest.cs the issue here is that you have the response be an object. 

So the correct fix is to remove the result object and just have the type be a string in https://github.com/microsoft/vscode-mssql/blob/main/src/models/contracts/connection.ts#L263

Reply:
Out of curiosity where did you get that original result object definition from? I'm surprised this ever worked given what I'm seeing.

Reply:
I was following a similar function listDatabases and I did not clearly see the STS result until I started debugging it. Not sure why it worked at some point and didn't. But it's fixed now.